gensrc/thrift/Types.thrift

```diff
@@ -120,6 +120,9 @@ struct TTypeNode {
 
     // only used for structs; has struct_fields.size() corresponding child types
     3: optional list<TStructField> struct_fields
+
+    // Marking which subfield will be used, this value will be set in FE. Used for MapType and StructType.
```

Comment:
how is this used for MapType

Reply:
Map will only use key value, so it's `selected_fields` length is two. `[0, 1]` marking that we only need value column.